netdata.spec.in

```diff
@@ -1,90 +1,100 @@
+#
+# Conditional build:
+%bcond_without	systemd		# systemd
+%bcond_without	nfacct		# build with nfacct plugin
+
 %if 0%{?fedora} || 0%{?rhel} >= 7
-%global do_systemd 1
 %else
-%global do_systemd 0
+%undefine	with_systemd
 %endif
 
 Summary:	Linux real time system monitoring, over the web
 Name:		@PACKAGE_NAME@
 Version:	@PACKAGE_RPM_VERSION@
 Release:	@PACKAGE_RPM_RELEASE@%{?release_suffix}%{?dist}
-License:	GPLv2+
-URL:		http://firehol.org
-Source:		%{name}-@PACKAGE_VERSION@.tar.bz2
-
+License:	GPL v3+
+Group:		Applications/System
+Source0:	https://github.com/firehol/netdata/archive/v%{version}/%{name}-%{version}.tar.gz
```

Comment:
yep. in that case you declare `Version: 1.0.2`, `Release: 0.alpha5.1` :)

and `1.0.2_alpha5` is valid for `Version` (does not contain `-`), but it's "newer" than `1.0.2`:

```
$ rpmvercmp 1.0.2_alpha5 1.0.2
1.0.2_alpha5 > 1.0.2
```